packages/status-js/src/client/activityCenter.ts

```diff
@@ -98,15 +115,49 @@ export class ActivityCenter {
     this.emitLatest()
   }
 
+  // todo?: also calls `clearChatNotifications` (on non-action items/notifications)?
+  // markAllAsRead = () => {}
+
+  // todo?: for example from chat with red bar in UI indicating start
+  // markChatNotificationsAsUnreadSince = (category, id) => {}
+
+  // todo?: rename to `clearChatNotifications`; separate (button) from `markAllAsRead`?
+  // todo?: merge with `removeChatNotifications`; together with `notification.type` condition
   /**
    * Removes all notifications.
    */
-  removeNotifications = () => {
-    this.#notifications.clear()
+  removeNotifications = (category: 'all' | 'mentions' | 'replies') => {
+    // todo?: clear all non-actionable notification too
+    if (category === 'all') {
+      for (const notification of this.#notifications) {
+        const { isMention, isReply } = notification
+
+        if (!(isMention || isReply)) {
+          continue
+        }
```

Comment:
notice: because `notifications` (state) includes "Unreads" category too, which is a category that the designs don't really work with, for indicating unread chats where new messages aren't mentions nor replies